collectd/conf.go

```diff
@@ -183,15 +192,15 @@ func validatedCollectdConfig(metrics *Metrics) (*collectdConf, error) {
 	}
 
 	// Validate Metrics.Receivers.
-	if len(metrics.Receivers) != 1 {
-		return nil, errors.New("exactly one metrics receiver with type 'hostmetrics' is required.")
+	if len(metrics.Receivers) > 1 {
+		return nil, errors.New(`At most one metrics receiver with type "hostmetrics" is allowed.`)
```

Comment:
In Go, errors start with lowercase letters.

Reply:
Fixed.